misk/src/main/kotlin/misk/client/ClientMetricsInterceptor.kt

```diff
@@ -66,6 +74,27 @@ class ClientMetricsInterceptor private constructor(
     return null
   }
 
+  private fun emitStatusCodeMetrics(
+    actionName: String,
+    elapsedMillis: Double,
+    readTrailers: Boolean,
+    response: Response
+  ) {
+    val grpcStatusHeader = if (readTrailers) {
+      /**
```

Comment:
This should be inline doc style // within a function

Reply:
Good catch, it was moved inside since it's only describing the trailers part. Updated it.